src/platform/functions/index.js

```diff
@@ -117,12 +126,12 @@ exports.scrapeEventbrite = functions
             .firestore()
             .collection('events')
             .doc('scraped-events')
-            .collection('toronto')
+            .collection(city)
             .add(event)
             .catch((err) => functions.logger.info(err));
         })();
       });
       functions.logger.info('Uploaded to firestore');
     });
     res.send(collectiveEventsArray);
-  });
+  });
```

Comment:
```suggestion
  });

```
let's keep the last blank line for styling purposes

Reply:
done!